### PR TITLE
Simulink build for GNU compiler on Linux systems

### PR DESCRIPTION
**Pull request by bjonkman**
**Feature or improvement description**
Added code to allow gnu/linux combinations for Simulink builds.

**Related issue, if one exists**
https://github.com/OpenFAST/openfast/issues/387 and others have used this to build with gnu/linux.

**Impacted areas of the software**
Simulink interface

**Test results, if applicable**
This doesn't change any code that affects the current tests.

### CODE REVIEW

**Review by andrew-platt — APPROVED** [review]
Looks good to me, especially since it is already in use by others.

File: modules/nwtc-library/CMakeLists.txt [hunk 1]
```diff
@@ -79,8 +79,8 @@ if(BUILD_OPENFAST_SIMULINK_API)
 
   if (APPLE OR UNIX)
     if (${CMAKE_Fortran_COMPILER_ID} STREQUAL "GNU")
-      # set(NWTC_SYS_FILE src/SysMatlabLinuxGnu.f90)
-      message(FATAL_ERROR "SIMULINK with GNU is not yet supported.")
+      set(NWTC_SYS_FILE src/SysMatlabLinuxGnu.f90)
+      # message(FATAL_ERROR "SIMULINK with GNU is not yet supported.")
```
Comment (rafmudaf) [inline modules/nwtc-library/CMakeLists.txt:83]: Since we are supporting this combination, can we remove this error message?

Comment (andrew-platt) [reply modules/nwtc-library/CMakeLists.txt:83]: Makes sense to me. No need to leave it as a comment.

Comment (rafmudaf) [reply modules/nwtc-library/CMakeLists.txt]: Thank you @bjonkman :)

Comment (bjonkman) [reply modules/nwtc-library/CMakeLists.txt]: done

**Review by rafmudaf — APPROVED** [review]

**Pull request merged**